it/topics/scripting_languages/JavaScript/README.md

```diff
@@ -0,0 +1,224 @@
+# JavaScript
+
+- [Che cos'è JavaScript?](#What-is-javascript)
+    - [Cosa puoi fare con JavaScript?](#What-can-you-do-with-javascript)
+    - [Perché imparare JavaScript - 5 buoni motivi](#why-learn-javascript --- 5-buoni motivi)
```

Comment:
Please use hyphens and not spaces otherwise the IDs are broke and the markdown is invalid. I see 5 links broken in the table of contents.

Reply:
If you want, use the VS Code Markdown All In One extension to regenerate the table of contents for you.